src/clientsettingsdlgbase.ui

```diff
@@ -674,7 +675,6 @@
   <tabstop>rbtBufferDelayPreferred</tabstop>
   <tabstop>rbtBufferDelayDefault</tabstop>
   <tabstop>rbtBufferDelaySafe</tabstop>
-  <tabstop>butDriverSetup</tabstop>
```

Comment:
@mulyaj I think we need to re-add this here. So you'd add 
```xml
 <tabstop>cbxSoundcard</tabstop>
 <tabstop>butDriverSetup</tabstop>
```

right under `<tabstops>`

Reply:
In fact, it seems to work without this too. Nevertheless, I will do some more tests.

Reply:
Please keep them.  They help accessibility tools, as far as I'm aware.

Reply:
I'm also quite sure that they should stay. @mulyaj or me can add them (but I'll not do it before tomorrow)

Reply:
I'll add them back